API/Controllers/OrdersController.cs

```diff
@@ -34,21 +34,52 @@ public OrdersController(IUnitOfWork unitOfWork)
         }
 
         [HttpGet]
+        [Authorize(Roles = "Administrator,Worker,Client")]
         public async Task<ActionResult> GetOrders()
         {
-            // admin wszystkie
-            // worker wolne do wziecia lub przypisane do siebie 
-            // klient tylko swoje
-            return Ok();
+            if ((await ordersRepo.GetAll()).Any())
```

Comment:
Generalnie ten if to strzał po wszystkie zamówienia tylko po to aby sprawdzić czy jakieś są. Nie trzeba tego sprawadzać, możesz zwracać od razu wyniki - w przypadku braku zamówień zostanie zwrócona pusta lista.

Reply:
Dla naszego projektu nie ma to znaczenia ale na przyszłość to cenna uwaga.